app/components/contributors/edit_component.html.erb

```diff
@@ -0,0 +1,41 @@
+
+<%= tag.div data: { controller: 'contributors' } do %>
+  <%= render Elements::Forms::ToggleComponent.new(form:, field_name: :role_type,
+                                                  label: t('works.edit.fields.contributors.role_type.label'),
+                                                  options: role_type_options) %>
+  <%= render Elements::Forms::SelectFieldComponent.new(form:, field_name: :person_role,
+                                                       options: PERSON_ROLES, label: t('works.edit.fields.contributors.role.label'),
+                                                       data: { contributors_target: 'selectPersonRole' }) %>
+  <%= render Elements::Forms::SelectFieldComponent.new(form:, field_name: :organization_role,
+                                                       options: ORGANIZATION_ROLES, label: t('works.edit.fields.contributors.role.label'),
+                                                       data: { contributors_target: 'selectOrganizationRole' }) %>
+
+  <!-- Contributor person name/orcid fields -->
+  <%= tag.div data: { contributors_target: 'personName' } do %>
```

Comment:
I personally find using `<%= tag.div data: { contributors_target: 'personName' } do %>` harder to read than `<div data-contribors-target="personName">` when HTML only (and no variable). It makes it clearer where the erb code is.

Reply:
Done.